src/components/Seo.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+
+import { SiteMetadata } from "./queries"
+
+const Seo = (props) => {
+  const data = SiteMetadata();
```

Comment:
Tutaj troszkę za dużo wyniosłeś, chodziło mi w komentarzu aby zostało coś takiego:
```
const data = useStaticQuery(getSiteMetaData);
```

Natomiast wewnątrz pliku `./queries` miałbyś wtedy
```
export const getSiteMetaData = graphql`
    query GetSiteMetadata {
      site {
        siteMetadata {
          description
          image
          siteUrl
          title
        }
      }
    }
  `
```

Reply:
niestety nie da się jako argumentu useStaticQuery podać zmiennej, dlatego rozwiązałem to w taki sposób, a query przynajmniej jest teraz w innym pliku. Daj znać co o tym sądzisz, czy może lepiej kombinować coś innego?

Reply:
Spoko, to jak się nie da to niech zostanie. I tak fajnie, że udało się wydzielić do osobnego pliku :D